scripts/scalafmt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# set -x
+
+HERE="`dirname $0`"
+VERSION="1.2.0"
+COURSIER="$HERE/.coursier"
+SCALAFMT="$HERE/.scalafmt-$VERSION"
+
+if [ ! -f $COURSIER ]; then
+  curl -L -o $COURSIER https://git.io/vgvpD
+  chmod +x $COURSIER
+fi
+
+if [ ! -f $SCALAFMT ]; then
+  $COURSIER bootstrap com.geirsson:scalafmt-cli_2.11:$VERSION --main org.scalafmt.cli.Cli -o $SCALAFMT
+  chmod +x $SCALAFMT
+fi
+
+$SCALAFMT "$@"
```

Comment:
Checks will not fail if Scala code mis-formatted.
It prints exception to console but exit code is 0

Reply:
See https://travis-ci.com/kornilova-l/scala-native-bindgen/jobs/128281617

Reply:
This should work:
```bash
exit $($SCALAFMT "$@" >/dev/null)
```